core/primitives-core/src/config.rs

```diff
@@ -72,6 +75,7 @@ pub struct VMLimitConfig {
     /// Max contract size
     pub max_contract_size: u64,
     /// Max transaction size
+    #[serde(default = "default_max_transaction_size")]
```

Comment:
is it strange that this is the only field that has a default set - please explain in the comments why

Reply:
This field was introduced in protocol version 46, but was not explicitly included into runtime configs.
This was not problematic because it didn't affect any of past txs, but actually I think it's more meaningful to remove default value and include this field into all past configs - what I did now.